packages/gestalt/src/TextField.css

```diff
@@ -9,17 +9,26 @@
   color: var(--g-colorGray200);
 }
 
-.unstyledTextField {
+.unstyledTextField, .measure {
```

Comment:
Ah, just need to update the class name here too

Reply:
good catch